### ISSUE TITLE
Update crop

### ISSUE THREAD

**Issue by InrcedibleHolg** [InrcedibleHolg/HerrSommer-1.13#6]
missing:
* potatoes
* carrots
* beetrot
* wheat

**Comment by InrcedibleHolg** [comment]
Keep the default blockstates and models, only changing the texture names.
* block/potatoes_stage1
* block/potatoes_stage2 
* block/potatoes_stage3

current names
* potatoes_stage_1.png
* potatoes_stage_2.png
* potatoes_stage_3.png

**Comment by InrcedibleHolg** [comment]
Texture rename for carrots:
* carrots_stage_0.png -> carrots_stage0.png

**Comment by InrcedibleHolg** [comment]
still missing:
* sugarcane
* melons
* pumpkin

**Comment by InrcedibleHolg** [comment]
Sugarcane:
```json
{
    "parent": "block/tinted_cross",
    "textures": {
        "cross": "block/sugar_cane"
    }
}
```

**Comment by InrcedibleHolg** [comment]
melon:
* unattached: rename melon_stem_disconnected.png to block/stem_growth0
Model for attached stem:
* blockstates/attached_melon_stem.json

**Comment by InrcedibleHolg** [comment]
Pumkin:
renamed:    pumpkin_stem_connected.png -> attached_pumpkin_stem.png
renamed:    pumpkin_stem_disconnected.png -> pumpkin_stem.png

**Comment by InrcedibleHolg** [comment]
Seeds:
* ./assets/minecraft/textures/item/seeds_melon.png
* ./assets/minecraft/textures/item/seeds_pumpkin.png
* ./assets/minecraft/textures/item/seeds_wheat.png
to 

./assets/minecraft/textures/item/wheat_seeds.png
./assets/minecraft/textures/item/pumpkin_seeds.png
./assets/minecraft/textures/item/melon_seeds.png
commit: e291b02

**Comment by InrcedibleHolg** [comment]
merged in master with e291b02
